### ISSUE TITLE
Fine-tuning OCR: training success on CPU but failed without any code error on GPU

### ISSUE THREAD

**Issue by Sayaka91** [PaddlePaddle/PaddleOCR#10192]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Windows 11
- 版本号/Version：Paddle：2.4.2  PaddleOCR：2.6.1.3 问题相关组件/Related components：OCR
- 运行指令/Command Code：
python tools/train.py -c configs/rec/PP-OCRv3/multi_language/japan_PP-OCRv3_rec.yml -o Global.pretrained_model=pretrain_models/japan_PP-OCRv3_rec_train/best_accuracy.pdparams
- 完整报错/Complete Error Message：
[2023/06/16 15:56:54] ppocr INFO: Architecture : 
[2023/06/16 15:56:54] ppocr INFO:     Backbone :
[2023/06/16 15:56:54] ppocr INFO:         last_conv_stride : [1, 2]
[2023/06/16 15:56:54] ppocr INFO:         last_pool_type : avg
[2023/06/16 15:56:54] ppocr INFO:         name : MobileNetV1Enhance
[2023/06/16 15:56:54] ppocr INFO:         scale : 0.5
[2023/06/16 15:56:54] ppocr INFO:     Head :
[2023/06/16 15:56:54] ppocr INFO:         head_list :
[2023/06/16 15:56:54] ppocr INFO:             CTCHead :
[2023/06/16 15:56:54] ppocr INFO:                 Head :
[2023/06/16 15:56:54] ppocr INFO:                     fc_decay : 1e-05
[2023/06/16 15:56:54] ppocr INFO:                 Neck :
[2023/06/16 15:56:54] ppocr INFO:                     depth : 2
[2023/06/16 15:56:54] ppocr INFO:                     dims : 64
[2023/06/16 15:56:54] ppocr INFO:                     hidden_dims : 120
[2023/06/16 15:56:54] ppocr INFO:                     name : svtr
[2023/06/16 15:56:54] ppocr INFO:                     use_guide : True
[2023/06/16 15:56:54] ppocr INFO:             SARHead :
[2023/06/16 15:56:54] ppocr INFO:                 enc_dim : 512
[2023/06/16 15:56:54] ppocr INFO:                 max_text_length : 25
[2023/06/16 15:56:54] ppocr INFO:         name : MultiHead
[2023/06/16 15:56:54] ppocr INFO:     Transform : None
[2023/06/16 15:56:54] ppocr INFO:     algorithm : SVTR
[2023/06/16 15:56:54] ppocr INFO:     model_type : rec
[2023/06/16 15:56:54] ppocr INFO: Eval :
[2023/06/16 15:56:54] ppocr INFO:     dataset :
[2023/06/16 15:56:54] ppocr INFO:         data_dir : ./train_data/finetune/eval/
[2023/06/16 15:56:54] ppocr INFO:         label_file_list : ['./train_data/finetune/eval.txt']
[2023/06/16 15:56:54] ppocr INFO:         name : SimpleDataSet
[2023/06/16 15:56:54] ppocr INFO:         transforms :
[2023/06/16 15:56:54] ppocr INFO:             DecodeImage :
[2023/06/16 15:56:54] ppocr INFO:                 channel_first : False
[2023/06/16 15:56:54] ppocr INFO:                 img_mode : BGR
[2023/06/16 15:56:54] ppocr INFO:             MultiLabelEncode : None
[2023/06/16 15:56:54] ppocr INFO:             RecResizeImg :
[2023/06/16 15:56:54] ppocr INFO:                 image_shape : [3, 48, 320]
[2023/06/16 15:56:54] ppocr INFO:             KeepKeys :
[2023/06/16 15:56:54] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2023/06/16 15:56:54] ppocr INFO:     loader :
[2023/06/16 15:56:54] ppocr INFO:         batch_size_per_card : 32
[2023/06/16 15:56:54] ppocr INFO:         drop_last : False
[2023/06/16 15:56:54] ppocr INFO:         num_workers : 1
[2023/06/16 15:56:54] ppocr INFO:         shuffle : False
[2023/06/16 15:56:54] ppocr INFO: Global :
[2023/06/16 15:56:54] ppocr INFO:     cal_metric_during_train : True
[2023/06/16 15:56:54] ppocr INFO:     character_dict_path : ppocr/utils/dict/japan_dict.txt
[2023/06/16 15:56:54] ppocr INFO:     checkpoints : None
[2023/06/16 15:56:54] ppocr INFO:     debug : False
[2023/06/16 15:56:54] ppocr INFO:     distributed : False
[2023/06/16 15:56:54] ppocr INFO:     epoch_num : 50
[2023/06/16 15:56:54] ppocr INFO:     eval_batch_step : [0, 50]
[2023/06/16 15:56:54] ppocr INFO:     infer_img : doc/imgs_words/ch/word_1.jpg
[2023/06/16 15:56:54] ppocr INFO:     infer_mode : False
[2023/06/16 15:56:54] ppocr INFO:     log_smooth_window : 20
[2023/06/16 15:56:54] ppocr INFO:     max_text_length : 25
[2023/06/16 15:56:54] ppocr INFO:     pretrained_model : pretrain_models/japan_PP-OCRv3_rec_train/best_accuracy.pdparams
[2023/06/16 15:56:54] ppocr INFO:     print_batch_step : 10
[2023/06/16 15:56:54] ppocr INFO:     save_epoch_step : 3
[2023/06/16 15:56:54] ppocr INFO:     save_inference_dir : None
[2023/06/16 15:56:54] ppocr INFO:     save_model_dir : ./output/v3_japan_mobile
[2023/06/16 15:56:54] ppocr INFO:     save_res_path : ./output/rec/predicts_ppocrv3_japan.txt
[2023/06/16 15:56:54] ppocr INFO:     use_gpu : True
[2023/06/16 15:56:54] ppocr INFO:     use_space_char : False
[2023/06/16 15:56:54] ppocr INFO:     use_visualdl : False
[2023/06/16 15:56:54] ppocr INFO: Loss :
[2023/06/16 15:56:54] ppocr INFO:     loss_config_list :
[2023/06/16 15:56:54] ppocr INFO:         CTCLoss : None
[2023/06/16 15:56:54] ppocr INFO:         SARLoss : None
[2023/06/16 15:56:54] ppocr INFO:     name : MultiLoss
[2023/06/16 15:56:54] ppocr INFO: Metric :
[2023/06/16 15:56:54] ppocr INFO:     ignore_space : False
[2023/06/16 15:56:54] ppocr INFO:     main_indicator : acc
[2023/06/16 15:56:54] ppocr INFO:     name : RecMetric
[2023/06/16 15:56:54] ppocr INFO: Optimizer :
[2023/06/16 15:56:54] ppocr INFO:     beta1 : 0.9
[2023/06/16 15:56:54] ppocr INFO:     beta2 : 0.999
[2023/06/16 15:56:54] ppocr INFO:     lr :
[2023/06/16 15:56:54] ppocr INFO:         learning_rate : 0.001
[2023/06/16 15:56:54] ppocr INFO:         name : Cosine
[2023/06/16 15:56:54] ppocr INFO:         warmup_epoch : 5
[2023/06/16 15:56:54] ppocr INFO:     name : Adam
[2023/06/16 15:56:54] ppocr INFO:     regularizer :
[2023/06/16 15:56:54] ppocr INFO:         factor : 3e-05
[2023/06/16 15:56:54] ppocr INFO:         name : L2
[2023/06/16 15:56:54] ppocr INFO: PostProcess :
[2023/06/16 15:56:54] ppocr INFO:     name : CTCLabelDecode
[2023/06/16 15:56:54] ppocr INFO: Train : 
[2023/06/16 15:56:54] ppocr INFO:     dataset :
[2023/06/16 15:56:54] ppocr INFO:         data_dir : ./train_data/finetune/train/
[2023/06/16 15:56:54] ppocr INFO:         ext_op_transform_idx : 1
[2023/06/16 15:56:54] ppocr INFO:         label_file_list : ['./train_data/finetune/train.txt']
[2023/06/16 15:56:54] ppocr INFO:         name : SimpleDataSet
[2023/06/16 15:56:54] ppocr INFO:         transforms :
[2023/06/16 15:56:54] ppocr INFO:             DecodeImage :
[2023/06/16 15:56:54] ppocr INFO:                 channel_first : False
[2023/06/16 15:56:54] ppocr INFO:                 img_mode : BGR
[2023/06/16 15:56:54] ppocr INFO:             RecConAug :
[2023/06/16 15:56:54] ppocr INFO:                 ext_data_num : 2
[2023/06/16 15:56:54] ppocr INFO:                 image_shape : [48, 320, 3]
[2023/06/16 15:56:54] ppocr INFO:                 prob : 0.5
[2023/06/16 15:56:54] ppocr INFO:             RecAug : None
[2023/06/16 15:56:54] ppocr INFO:             MultiLabelEncode : None
[2023/06/16 15:56:54] ppocr INFO:             RecResizeImg :
[2023/06/16 15:56:54] ppocr INFO:                 image_shape : [3, 48, 320]
[2023/06/16 15:56:54] ppocr INFO:             KeepKeys :
[2023/06/16 15:56:54] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2023/06/16 15:56:54] ppocr INFO:     loader :
[2023/06/16 15:56:54] ppocr INFO:         batch_size_per_card : 128
[2023/06/16 15:56:54] ppocr INFO:         drop_last : True
[2023/06/16 15:56:54] ppocr INFO:         num_workers : 4
[2023/06/16 15:56:54] ppocr INFO:         shuffle : True
[2023/06/16 15:56:54] ppocr INFO: profiler_options : None
[2023/06/16 15:56:54] ppocr INFO: train with paddle 2.4.2 and device Place(gpu:0)
[2023/06/16 15:56:54] ppocr INFO: Initialize indexs of datasets:['./train_data/finetune/train.txt']
[2023/06/16 15:56:54] ppocr INFO: Initialize indexs of datasets:['./train_data/finetune/eval.txt']
W0616 15:56:54.185643  2312 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 12.1, Runtime API Version: 10.2
W0616 15:56:54.187649  2312 gpu_resources.cc:91] device: 0, cuDNN Version: 8.9.

I tried to finetuning the text recognition model with my custom dataset. 
This works well on CPU but when I use GPU, the training processing exited without any error code.

**Comment by Sayaka91** [comment]
This problem happens because CUDA version and CUDNN version used in paddlepaddle are incompatible with its version in my device. 
Just reinstall CUDA version 10.2 for paddlepaddle 2.4.2

**Comment by kwanxian** [comment]
看文档：https://www.paddlepaddle.org.cn/documentation/docs/zh/install/pip/windows-pip.html
